docs/referece_template_client_libraries.md

```diff
@@ -0,0 +1,56 @@
+# Method name
```

Comment:
Perhaps we should lower headings by 1 or 2 levels, because ` h1` should be the package name. Maybe we want to add `Table of contents` and `Reference`  headings too...